implementation/orchestration-to-choreography/README.md

```diff
@@ -0,0 +1,90 @@
+# Convert Orchestration to Choreography
+This project is the CDK implementation of ['Convert Orchestration to Choreography'](https://serverlessland.com/refactoring-serverless/orchestration-to-choreography) pattern. It shows how you can convert the interaction approach from orchestration to choreography without changing the functionality.
+
+## How it works
+The scenario is to simulate a mortgage enquiry use case. The broker receives an enquiry from a loan applicant, and then distributes the request to a number of banks. After receiving the offers from the banks, the broker will aggregate the results and return to the applicant. 
+
+In the orchestration solution, a Step Functions State Machine is used to coordinate the interactions between a number of tasks which involve the different parties, and ultimately complete the workflow. In the choreography solution, there is no central coordinator to facilitate the interactions, and the parties don't talk to each other directly. They actually don't know or even don't care about the existence of other parties. Each party simply publishes the message to SNS topic or SQS queue, and the other parties will subscribe and consume the message from the topic or the queue if they are interested in this type of the message. In this way, a loose-coupling architecture is achieved since all the involved parties don't need to know the existence of others and there is no direct communication between one another.
+
+The code will deploy 2 CloudFormation stacks:
+- OrchestrationStack: Here is the State Machine that invokes a number of Lambda functions, coordinates the communication, and completes the workflow.
+- ChoreographyStack: Here are a number of tasks (in the form of Lambda function), which operate independently without coupling with each other. They use Publisher/Subscriber pattern and message to queue to work together asynchronously and ultimately achieve the same result as an explicit workflow.
+
+---
+## Deploy the infrastructure
+
+
+To build this app, navigate to `implementation/orchestration-to-choreography` folder. Then run the following:
+
+```bash
+npm install -g aws-cdk
+npm install
+```
+
+This will install the necessary CDK, dependencies, build your TypeScript files and CloudFormation template.
+
+Next, deploy the 2 Stacks to your AWS Account (`OrchestrationStack` and `ChoreographyStack`):
+``` 
+cdk deploy --all
```

Comment:
cdk deploy is failing for me. See screenshot below. 

<img width="992" alt="Screenshot 2023-06-21 at 2 18 58 PM" src="https://github.com/aws-samples/aws-refactoring-to-serverless/assets/124315783/4da707da-cdf6-44d6-8233-61b167dc2fcf">

Reply:
Fixed the Python version problem